nilearn/plotting/find_cuts.py

```diff
@@ -64,13 +64,19 @@ def find_xyz_cut_coords(img, mask=None, activation_threshold=None):
         if mask is None:
             mask = not_mask
         else:
-            mask *= not_mask
+            if isinstance(mask, np.ndarray):
+                mask *= not_mask
+            else:
+                mask = check_niimg_3d(mask)
```

Comment:
We should be doing this on the top. First, checking if it is a valid Nifti image in 3D and second using _safe_get_data on mask image to convert to data numpy array. Rest should do the job by itself. Tell me if I miss something here ?

 I think this snippet should be added on top after ```data = _safe_get_data(img)``` at line 55.
```python
mask_img=check_niimg_3d(mask_img)
mask = _safe_get_data(mask_img)
```